influxql/ast.go

```diff
@@ -472,6 +472,7 @@ func IsSystemName(name string) bool {
 	case "_fieldKeys",
 		"_measurements",
 		"_series",
+		"_seriesKey",
```

Comment:
Yes.  Because its called from here: https://github.com/influxdata/influxdb/pull/8691/files#diff-0cae4acf5479b75328bd79505447da9eR980 and to fix `TestServer_Query_ShowSeriesCardinality`.